buildbot/configure.py

```diff
@@ -82,6 +82,10 @@ def do_configure(args, passthrough_args):
     if libclc_enabled:
         llvm_enable_projects += ";libclc"
 
+    # DeviceRIL uses -fuse-ld=lld, so enbale lld.
```

Comment:
```suggestion
    # DeviceRIL uses -fuse-ld=lld, so enable lld.
```

What is "DeviceRIL"? [Search](https://github.com/search?q=repo%3Aintel%2Fllvm%20DeviceRIL&type=code) over this project didn't find anything.